File: C/Estrutura Repetitiva/Para/Divisao.c
#include <stdio.h>
#include <string.h>


int main ()
{
    int N, i;
    double numerador, denominador,divisao;

    printf ("Quantos casos voce vai digitar? ");
    scanf ("%d", &N);


    for (i = 0; i < N; i++)
    {
        printf ("Entre com o numerador: ");
        scanf ("%lf", &numerador);
        printf ("Entre com o denominador: ");
        scanf ("%lf", &denominador);

        if (numerador == 0)
        {
            printf ("DIVISAO = 0.00\n");

        }
        else if (denominador == 0)
        {
            printf ("DIVISAO IMPOSSIVEL\n");

        }
        else
        {
            divisao = numerador / denominador;
            printf ("DIVISAO = %.2lf\n", divisao);

        }
    }


    return 0;
}

File: Java/Estrutura repetitiva/Para/Divisao.java
import java.util.Locale;
import java.util.Scanner;


public class Divisao {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int i, N, a, b;
        double divisao;

        System.out.print("Quantos casos voce vai digitar? ");
        N = sc.nextInt();

        for (i = 1; i <= N; i++) {
            System.out.print("Entre com o numerador: ");
            a = sc.nextInt();
            System.out.print("Entre com o denominador: ");
            b = sc.nextInt();
            if (b == 0) {
                System.out.println("DIVISAO IMPOSSIVEL");
            }
            else {
                divisao = (double)a/b;
                System.out.println("DIVISAO = " + String.format("%.2f", divisao));
            }

        }

        sc.close();

    }
}
